src/components/NavBar/styles.css

```diff
@@ -0,0 +1,5 @@
+.NavBar{
```

Comment:
minor but prefer classnames be lowercase, so this would be either `.navbar` or `.nav-bar`. But that classname could be potentially conflicting with other classes from bootstrap/material.

If we're going to stick with bootstrap, which I _think_ you can use together with material (you really shouldn't though, but it's fine for now) -> it might be easier just to add `.d-flex justify-between w-100` classes on the navbar itself.